frontend/public/components/pod.tsx

```diff
@@ -758,6 +758,8 @@ export const PodDetailsList: React.FC<PodDetailsListProps> = ({ pod }) => {
         </DetailsItem>
       )}
       <RuntimeClass obj={pod} path="spec.runtimeClassName" />
+      <RuntimeClass obj={pod} />
```

Comment:
> This would be a little odd if they both were successfully rendered ... as `<RuntimeClass />` has some the same label: `label={t('public~Runtime class')}`. What are the odds of both `obj.spec.template.spec.runtimeClassName` and `obj.spec.runtimeClassName` being in the same object?

Oops! Duplicate.  Good catch! Thanks.